### PR TITLE
Update rat sh

### PR DESCRIPTION
**Pull request by kruzikh**

### CODE REVIEW

File: valuesets/source-update/update-rat.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+curl https://covid-19-diagnostics.jrc.ec.europa.eu/devices/hsc-common-recognition-rat |
```
Comment (gabywh) [inline valuesets/source-update/update-rat.sh:1]: `curl --silent --show-error http...`

Comment (gabywh) [inline valuesets/source-update/update-rat.sh:1]: 1st line start with #!/bin/sh , curl on 2nd line :
```sh
#!/bin/sh
curl --silent --show-error http
```

File: valuesets/source-update/update-rat.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+jq '.extracted_on as $version | .deviceList | reduce .[] as $i
+({};
+.[$i.id_device] =
+{ "display": ($i.manufacturer.name + ", " + $i.commercial_name),
+    "lang":"en",
+    "active": true,
+    "system": "https://covid-19-diagnostics.jrc.ec.europa.eu/devices",
+    "version": $i.last_updated
+}) | {"valueSetId": "covid-19-lab-test-manufacturer-and-name", "valueSetDate":$version[0:10], "valueSetValues": .}' > ~/dev/GitHub/ehn-dgc-schema/valuesets/test-manf.json
```
Comment (gabywh) [inline valuesets/source-update/update-rat.sh:10]: Remove output redirect from script ( `> ~/ ...`) and allow caller to pipe / redirect at shell level, if required.

**Review by gabywh — CHANGES_REQUESTED** [review]

File: valuesets/source-update/update-rat.sh (hunk 2 above)
Comment (kruzikh) [reply valuesets/source-update/update-rat.sh:10]: Would it be ok, to mark it as a comment?

#> ~/dev/GitHub/ehn-dgc-schema/valuesets/test-manf.json

Comment (jschlyter) [reply valuesets/source-update/update-rat.sh:10]: I would remove it and create a makefile target that uses this script ot update the in-repository files

Comment (kruzikh) [reply valuesets/source-update/update-rat.sh:10]: Guys, could you do it? I am afraid that this goes beyond my programming experience.

Comment (gabywh) [reply valuesets/source-update/update-rat.sh:10]: yes, will modify in-situ, don't need a makefile yet - T-Systems will basically take this script and use it for updates, up to them how they embed it

**Review by ryanbnl — APPROVED** [review]
Nice, even works under WSL2 ;)

**Pull request merged**